src/ethereum-indexed-data/service-request/service-request.service.ts

```diff
@@ -177,13 +177,17 @@ export class ServiceRequestService {
     }
 
     const result = [];
-    const requestServiceByCustomers = await this.elasticsearchService.search(
-      searchObj,
-    );
-
-    requestServiceByCustomers.body.hits.hits.forEach((requestService) => {
-      result.push(requestService._source);
-    });
+    try {
+      const requestServiceByCustomers = await this.elasticsearchService.search(
+        searchObj,
+      );
+  
+      requestServiceByCustomers.body.hits.hits.forEach((requestService) => {
+        result.push(requestService._source);
+      });
+    } catch (error) {    
+      console.log('API "service-requests/customer/:customerId":', error.body.error.reason);
```

Comment:
Instead of `console.log` use Loggers instead.

https://docs.nestjs.com/techniques/logger